File: code/pete/javascript/examples/python-to-js/grading/grading.js
let percentage = prompt('enter a percentage grade')

// console.log('hello')

// console.log(typeof percentage)

percentage = parseInt(percentage) // python equivalent percentage = int(percentage)


// console.log(typeof percentage)

let grade

if (percentage >= 90) {
	grade = 'A'
}
else if (percentage >= 80) {
	grade = 'B'
}
else if (percentage >= 70) {
	grade = 'C'
}
else if (percentage >= 60) {
	grade = 'D'
}
else {
	grade = 'F'
}

// alert('your grade is ' + grade)
alert(`your grade is: ${grade} ${percentage}`) // template literal, aka string interpolation, aka javascript f-string


let var1 = 1
let var2 = '2'
console.log(var1 + var2)

// let, const, var

// let number = 1
// alert(number)
// number += 1
// alert(number)

// const is a constant variable, cannot be redefined
// const number2 = 1
// alert(number2)
// number2 += 1 // cannot redefine constant variable
// alert(number2)

// const numbers = [1,2,3,4,5]
// numbers.push(6)
// numbers = [1, 2, 3, 4, 5, 6] // cannot redefine constant variable
// alert(numbers)

File: code/pete/javascript/examples/python-to-js/grading/grading.py
# prompt: give me a percentage grade
# user types in 85
# program says they got a B

percentage = int(input('enter a percentage grade: '))

if percentage >= 90:
    grade = 'A'
elif percentage >= 80:
    grade = 'B'
elif percentage >= 70:
    grade = 'C'
elif percentage >= 60:
    grade = 'D'
else:
    grade = 'F'

print(f"You got a{'n' if grade == 'A' else ''} {grade}.")

# number = 2
# # python oneline conditional
# even_or_odd = 'even' if number % 2 == 0 else 'odd'

# if number % 2 == 0:
#     even_or_odd = 'even'
# else:
#     even_or_odd = 'false'

# def add(a, b):
#     return a + b

# print(a, b)


var1 = 1
var2 = '2'
print(var1 + var2)